src/lib/draw/GeometryMode.svelte

```diff
@@ -123,21 +126,22 @@
   function startEditing(id: number) {
     mapHover.set(null);
 
-    let feature;
+    let maybeFeature: FeatureUnion | null = null;
```

Comment:
I tried leaving off the `= null` part entirely, but then got weird errors about it being uninitialized when we use it; maybe it didn't understand the store.update bit